deploy/006-deploy-psm.ts

```diff
@@ -1,48 +1,88 @@
+import deployOracle from "@venusprotocol/oracle/dist/deploy/1-deploy-oracles";
 import { parseUnits } from "ethers/lib/utils";
 import { ethers } from "hardhat";
 import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
-import ADDRESSES from "../helpers/address";
-
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  const { deployments, network, getNamedAccounts } = hre;
+  const { deployments, network, getNamedAccounts, getChainId } = hre;
   const { deploy } = deployments;
   const { deployer } = await getNamedAccounts();
 
-  const networkName = network.name === "bscmainnet" ? "bscmainnet" : "bsctestnet";
-  const addresses = ADDRESSES[networkName];
+  // Ensure oracles are deployed
+  await deployOracle(hre);
+
+  const usdtAddress = (
+    await deploy("USDT", {
+      contract: "MockToken",
+      from: deployer,
+      args: ["Tether", "USDT", 18],
+      log: true,
+      autoMine: true,
+    })
+  ).address;
+  const acmAddress = (await deployments.get("AccessControlManager")).address;
+  const treasuryAddress = (await deployments.get("VTreasury")).address;
+  const oracleAddress = (await deployments.get("ResilientOracle")).address;
+
+  let normalVipTimelockAddress;
+  if (hre.network.name === "hardhat") {
+    normalVipTimelockAddress = (
+      await deploy("NormalTimelock", {
+        contract: "TestTimelockV8",
+        from: deployer,
+        args: [deployer, 600],
+        log: true,
+        autoMine: true,
+      })
+    ).address;
+  } else {
+    // We should be able to fetch the deployed contract from the governance network on live networks
+    normalVipTimelockAddress = (await deployments.get("NormalTimelock")).address;
+  }
+
+  const vaiAddress = (
+    await deploy("VAI", {
+      from: deployer,
+      args: [await getChainId()],
+      log: true,
+      autoMine: true,
+    })
+  ).address;
```

Comment:
I think this deployment should be conditional. I don't think we should deploy the VAI contract on a live network

Reply:
VAI is already deployed, so, no redeployment should happen in a live network